nupic/research/frameworks/vernon/experiments/meta_cl_experiment.py

```diff
@@ -203,6 +207,8 @@ def compute_class_indices(cls, config, dataset, mode="all", sample_size=None):
         for idx, (_, target) in enumerate(dataset):
             if isinstance(target, torch.Tensor):
                 target = target.item()
+            elif isinstance(target, np.integer):
+                target = int(target)
```

Comment:
Just curious, any specific cases where this requirement came up? I would expect datasets loaded by pytorch to use tensor format.

Reply:
Our GSC preprocessed dataset used numpy.int64 types, not sure why. It seems it's possible for the targets to be int, torch.int, or numpy.int